lib/test.js

```diff
@@ -106,8 +107,11 @@ Test.prototype.test = function (name, opts, cb) {
     });
 };
 
-Test.prototype.comment = function (msg) {
+Test.prototype.comment = function () {
     var that = this;
+    var keys = Object.keys(arguments)
+    var msg = keys.length > 1 ? format.apply(null, arguments) : arguments[keys[0]];
```

Comment:
Calling `format.apply` in all cases changes the behavior, e.g. `test.comment({answer:42})` will print `{answer:42}` whereas the current code prints `[object Object]`. I'm happy to make the change, as I think that's preferable behavior, but it's up to you.